### PR TITLE
heat-205 Add elasticache alerts

### PR DESCRIPTION
**Pull request by babuharidass11**

### CODE REVIEW

File: charts/generic-prometheus-alerts/templates/aws-elasticache-alerts.yaml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+        {{- range $key, $val := $.Values.elastiCacheAlertsClusterIds }}
+        - alert: elasticache-enginecpu-utilisation
+          annotations:
+            message: elasticache cluster {{ $val }} in {{ $targetNamespace }} - CPU utilisation at {{`{{`}} printf "%.2f" $value {{`}}`}} which is over threshold of {{ $elastiCacheAlertsEngineCPUThreshold }} for last {{ $elastiCacheAlertsEngineCPUThresholdMinutes }} mins. This alert configured by app {{ $targetNamespace }}/{{ $targetApplication }}.
+            runbook_url: {{ $.Values.runbookUrl }}elasticache-enginecpu-utilisation
+            {{/* TODO*/}}
+            dashboard_url: {{ $.Values.grafanaUrl }}/d/VR46pmwWk/aws-elasticache?orgId=1&var-datasource=Cloudwatch&var-region=default&var-elasticacheclusteid={{ $key }}&from=now-6h&to=now
+            summary: elasticache cluster CPU usage too high
+          expr: |-
+            aws_elasticache_engine_cpuutilization_maximum{cache_cluste_id='{{ $key }}'} > {{ $elastiCacheAlertsEngineCPUThreshold }}
```
Comment (mattops) [inline charts/generic-prometheus-alerts/templates/aws-elasticache-alerts.yaml:32]: `cache_cluste_id`  spelling error and see other occurances.

File: charts/generic-prometheus-alerts/values.yaml [hunk 2]
```diff
@@ -70,6 +70,28 @@ additionalRuleLabels: {}
...
+elastiCacheAlertsEngineCPUThresholdMinutes: 5
+
+# Set the alert for Elasticache's CPU threshold - value between 0 and 100.
+elastiCacheAlertsCPUThreshold: 90
+
+# Set the alert for Elasticache's CPU threshold (minutes).
+elastiCacheAlertsCPUThresholdMinutes: 5
+
+# Set the alert for Elasticache's FreeMemory threshold - value between 0 and 100.
+elastiCacheAlertsFreeMemoryThreshold: 150
```
Comment (mattops) [inline charts/generic-prometheus-alerts/values.yaml:90]: should this be between 0-100?

File: charts/generic-prometheus-alerts/templates/aws-elasticache-alerts.yaml (hunk 1 above)
Comment (babuharidass11) [reply charts/generic-prometheus-alerts/templates/aws-elasticache-alerts.yaml]: updated the typo error

File: charts/generic-prometheus-alerts/values.yaml (hunk 2 above)
Comment (babuharidass11) [reply charts/generic-prometheus-alerts/values.yaml:90]: updated the description

**Review by mattops — APPROVED** [review]

**Pull request merged**